recipes/ecco_access/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "ecco_access" %}
+{% set version = "0.2.1" %}
+{% set python_min = "3.7" %}
```

Comment:
Only set `python_min` if the minimum required version is higher than the conda-forge default.

```suggestion
```